lib/trace_mgr.js

```diff
@@ -353,7 +353,7 @@ function checkGlobalShouldEmit(emitKind, optInfo) {
     } else {
       return (emitOptions.emitOnExit === 'error') && (optInfo !== 0);
     }
-  } else if (emitKind == 'emitOnException' || emitKind == 'emitOnSigInt') {
+  } else if (emitKind === 'emitOnException' || emitKind === 'emitOnSigInt') {
```

Comment:
Kyle already fixed it in https://github.com/nodejs/node-chakracore/commit/4751de59621ce28f42a4a59f718308c681acaaff#diff-d3e8e0bdb7af6cb1b0059f4d2d23e4b1